modules/_node-scripts/configuration/getGlobalSymbols.mjs

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+
+import getRootDir from '../util/getRootDir.mjs';
+import require from '../util/require.mjs';
+
+/**
+ * @returns
+ * Something like:
+ *
+ * {
+ *   '@clayui/charts': ['__esModule', 'bb', 'default']
+ * }
+ */
+export default async function getGlobalSymbols() {
```

Comment:
nit: "global" can be a bit odd to me since that can mean different things in different contexts. What if we named it `getConfigSymbols` or `getConfigDefinedSymbols`?

idk if those are any better though

Reply:
Let's call it `overridenPackageSymbols` since it's what they are...

Reply:
done